msteams-platform/toolkit/debug-your-Teams-app-test-tool.md

```diff
@@ -112,6 +114,105 @@ Test Tool offers a faster debug experience for bot applications when compared to
+
+    ```bash
+    npm install -g @microsoft/teams-app-test-tool
+    ```
+
+1. Use the `teamsapp` CLI to create your first project. Start from the folder where you want to create the project folder.
+
+    ```bash
+    teamsapp new   
+    ```
+
+    You can use the CLI to create a new Teams app. The CLI leads you through a series of questions.
+
+    For example, use arrow keys to select an option. After you make the choice, select **Enter** to confirm:
+
+      1. Select **New Project**.
+      1. Select **Bot**.
+      1. Select **Basic Bot** as the app features.
+      1. Select **JavaScript** as the programming language.
+      1. Select **Enter** to select the default workspace folder.
+      1. Enter a suitable name for your app, like `HelloBot`. The name of the app must consist only of alphanumeric characters.
+
+    After you've answered all questions, your project is created.
+
+1. Install required dependencies and npm packages.
+
+    ```bash
+    teamsapp deploy --env=testtool
+    ```
+
+1. Start your bot app:
+
+    ```bash
+    npm run dev:teamsapp:testtool
+    ```
+
+1. In a separate terminal, initiate the Teams App Test Tool:
+
+    ```bash
+    npm run dev:teamsapp:launch-testtool
+    ```
+
+1. Test Tool opens the bot in a webpage.
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/test-tool.png" lightbox="../assets/images/teams-toolkit-v2/debug/test-tool.png" alt-text="Screenshot shows the bot open in Test Tool."
+
+# [Command line in C#](#tab/clicsharp)
+
+Run the following commands in the directory that contains `teamsapp.yml` file:
+
+1. Download the Teams App Test Tool CLI from the [GitHub](https://github.com/OfficeDev/TeamsFx/releases?q=teams-app-test-tool&expanded=true) release.
+
+1. Unzip the downloaded package to a folder. You'll find an executable binary file `teamsapptester.exe`.
+
+1. Run the following command to launch profile:
+
+    ```bash
+    dotnet run --launch-profile "Teams App Test Tool (browser)"
+    ```
+
+1. In a separate terminal, define your bot message endpoint:
+
+    1. For Command Prompt:
+
+        ```bash
+        set BOT_ENDPOINT=http://127.0.0.1:5130/api/messages
+        ```
+
+    1. For PowerShell:
+
+        ```bash
+        $env:BOT_ENDPOINT = "http://127.0.0.1:5130/api/messages"
+        ```
+
+1. Run the following command to initiate Teams App Test Tool:
+
+    1. For Command Prompt:
+
+        ```bash
+        teamsapptester.exe start
+        ```
+
+    1. For PowerShell:
+
+        ```bash
+        teamsapptester.exe start
+        ```
+
+1. Test Tool opens the bot in a webpage.
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/test-tool.png" lightbox="../assets/images/teams-toolkit-v2/debug/test-tool.png" alt-text="Screenshot shows the bot open in Test Tool."
+
+In case the Test Tool doesn't initiate because of a port conflict, alter the Test Tool's port number in the `TEAMSAPPTESTER_PORT` environment variable.
```

Comment:
In case the Test Tool doesn't initiate because of a port conflict, alter the Test Tool's port number in the `TEAMSAPPTESTER_PORT` environment variable in the terminal where you run `teamsapptester.exe start`.